__tests__/app.test.js

```diff
@@ -15,4 +25,27 @@ describe("GET /api", () => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
+});
+
+describe("GET /api/topics", () => {
+  test("200: Responds with a status code", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
```

Comment:
We are testing for the 200 status code our test below! We are still following the TDD principles when we say the `expect(200).then({body})` as the behavior is that the (single) response has both a status code _and_ in this case our topics, so we're safe to combine these 2 tests.